features/other_b/skin_wizard.feature

```diff
@@ -12,6 +12,32 @@ Feature: Skin wizard
   Then I should see "Create New Skin"
     And I should see "CSS"
 
+  @javascript
+  Scenario: User can add a parent skin to the wizard
```

Comment:
For clarity, maybe "User can add a parent skin using the wizard" for this and "User can add a parent skin using the Custom CSS form" for the other test?